packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -356,6 +356,9 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
    * set tenantId and subscriptionId
    */
   async setSubscription(subscriptionId: string): Promise<void> {
+    if (isV3Enabled()) {
+      return;
```

Comment:
Is this a workaround to make current v3 template work, or a final solution that V3 totally does not need `setSubscription`?

Reply:
Since there will be no "subscriptionInfo.json" in V3, our toolkit will not store the sub info anymore. So, I think `setSubscription` will be deprecated in V3. Users will only need to choose subscription before provision.